### ISSUE TITLE
i7 - not installing iSuperSU

### ISSUE THREAD

**Issue by tb205gti** [jakeajames/rootlessJB3#147]
iPhone 7 
iOS 1.2.1.1
Unable to install iSuperSU.

log snippet:
Successfully injected [1/1] to trust cache.
[-] unable to get vnode from path for /var/LIB/MobileSubstrate/DynamicLibraries/AppSyncUnified.dylib
[-] Unable to fix mmap of path: /var/LIB/MobileSubstrate/DynamicLibraries/AppSyncUnified.dylib
[*] Installing app com.jakeashacks.iSuperSU
[-] Can't install app /var/containers/Bundle/tweaksupport/Applications/iSuperSU.app
	[+] Really jailbroken!

Jailbroken, and tweaks working, just no iSuperSU. Have never worked on any of the commits.

Can I provide more logs to assist with the debugging?

**Comment by tb205gti** [comment]
Tried to run the steps manually - everything goes fine, apart from uicache.. It deletes the iSuperSU.app and fails with 
[*] Installing app com.jakeashacks.iSuperSU
[-] Can't install app /var/containers/Bundle/tweaksupport/Applications/iSuperSU.app

**Comment by jakeajames** [comment]
Does this always happen? What other rootlessJB versions used before? Did you mess with any of the files in Filza/ssh?

**Comment by tb205gti** [comment]
Yes, always - have never worked on this iPhone. my iPhone6 works - but it has never worked on this one. I have been using the different commits since the first one. Not messing with the files via ssh - have never used filza.

**Comment by johacko** [comment]
you have try to install apps with dpkg or uicache with ssh.
So you need to connect you to your iPhone with SSH en remove all files and folder in /var/Apps
with rm -rf /var/Apps/*

sorry for my bad english

**Comment by tb205gti** [comment]
If My interest is to fix the underlying issue that *might* be with rootlessJB - then I should not try all sorts of stuff on my own. I need to provide jake with enough debug logging for him to see if this is related to the JB or to my phone.

**Comment by tb205gti** [comment]
@jakeajames Any thoughts? Can I provide more logs?

Microsoft Outlook detects the JB when in JB state - I suppose iSuperSu can be used to prevent Outlook from discovering the JB?

**Comment by jakeajames** [comment]
NoSubstitute or blacklist it @tb205gti

**Comment by tb205gti** [comment]
blacklist works perfectly!

**Comment by hagomoro333** [comment]
i have problem with rootlessjb.. i tried to click 170 time Jailbreak and unjailbreak.. exploit failed and reboot..
